packages/vue/README.md

```diff
@@ -168,6 +171,29 @@ An example for calling such a method in a Vue component is shown here:
 </template>
 ```
 
+### 7.2. Obtaining a reference to the underlying native element
```

Comment:
Same as the comment for the angular readme: 

Would you remove the complete section 7.1 with the major update? Because I was thinking about there should be something with use methods in the title itself, so it can be found quickly by the users if they try to find out how to use methods.

It he other section is completely removed maybe the title could be something like:
Obtaining a reference to the underlying native element for e.g. method usage